src/main/java/com/calculator/string/StringCalculator.java

```diff
@@ -0,0 +1,40 @@
+package com.calculator.string;
+
+import com.calculator.operator.Operator;
+
+import java.util.Arrays;
+import java.util.Iterator;
+
+public class StringCalculator {
+
+    public static final String CALCULATION_FORMULA_SEPARATOR = " ";
+
+    /**
+     * 사칙연산 계산식을 수행하는 메소드 (입력 값에 따라 계산 순서가 결정된다)
+     *
+     * @param calculationFormula 숫자와 사칙 연산 사이에 빈 공백이 포함된 사칙연산 계산식 문자열
+     * @return 사칙연산 int 결과값
+     */
+    public static int calculate(String calculationFormula) {
+        isCaluationFormulaValid(calculationFormula);
```

Comment:
```suggestion
        isValidCaluationFormula(calculationFormula);
```

가끔 라이브러리나 다른 코드를 보면 isValid... 의 메서드 명이 좀 더 많은 것 같아서 제안드려봅니다 :)

Reply:
좋은 의견 감사합니다! 내용 반영해보겠습니다👍